CIPs/0008.md

```diff
@@ -2,15 +2,16 @@
 
 - Date: 2020-05-05
 - Author: @nambrot
```

Comment:
hi @nambrot  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo.